app/uk/gov/hmrc/merchandiseinbaggage/views/PurchaseDetailsExportView.scala.html

```diff
@@ -46,6 +46,8 @@ <h1 class="govuk-fieldset__legend govuk-fieldset__legend--xl">
             <label for="price">@messages("purchaseDetails.heading", goodsCategory)</label>
         </h1>
 
+        <div class="govuk-inset-text">@messages("purchaseDetails.type.of.goods")<strong> @goodsCategory</strong></div>
```

Comment:
dont we need message keys for this key `purchaseDetails.type.of.goods`? or is it already there ?

Reply:
is all there...yesterday i forgot to do export thats all.....